engine/src/main/java/org/terasology/world/WorldProvider.java

```diff
@@ -128,7 +129,7 @@
      * @return The named extra-data value at the given position
      */
     int getExtraData(String fieldName, Vector3i pos);
-    
+
     /**
      * Sets one of the per-block custom data values at the given position, if it is within the view.
      *
```

Comment:
```suggestion
     * Do not use this method for generating the world, use the overload taking chunk instead.
     *
```

Reply:
Also, consider adding similar overloads for getExtraData, and similar javadocs notices for them. My wording is a bit different to yours, pick whichever you like more, or ask for input from someone else.